test/test_nn.py

```diff
@@ -8843,6 +8843,15 @@ def test_batchnorm_non_contig_cpu(self):
         self.assertTrue(ref_out.is_contiguous())
         self.assertEqual(out, ref_out)
 
+        input_bf = torch.arange(24, dtype=torch.bfloat16).reshape(1, 3, 2, 4).cpu()
```

Comment:
Hmm, why CPU is needed here?
```suggestion
        input_bf = torch.arange(24, dtype=torch.bfloat16).reshape(1, 3, 2, 4)
```